ISL.Providers.Notifications.GovukNotify/Services/Foundations/Notifications/NotificationService.cs

```diff
@@ -85,11 +85,11 @@ public ValueTask<string> SendSmsAsync(
 
         public ValueTask<string> SendLetterAsync(
             string templateId,
-            Dictionary<string, dynamic> personalisation = null,
+            Dictionary<string, dynamic> personalisation,
             string clientReference = null) =>
             TryCatch(async () =>
             {
-                ValidateOnSendLetter(templateId);
+                ValidateOnSendLetter(templateId, personalisation);
 
                 return await this.govukNotifyBroker.SendLetterAsync(
                     templateId,
```

Comment:
@davidhayes03 / @JParkesAD  not sure if this was lost in translation, but we need to update this signature as discussed with DH.

```suggestion
        public ValueTask<string> SendLetterAsync(
            string templateId,
            string addressLine1,
            string addressLine2,
            string addressLine3,
            string addressLine4,
            string addressLine5,
            string addressLine6,
            string addressLine7,
            Dictionary<string, dynamic> personalisation,
            string clientReference = null) =>
            TryCatch(async () =>
            {
                ValidateOnSendLetter(templateId, personalisation);

                return await this.govukNotifyBroker.SendLetterAsync(
                    templateId,
```